dora/core/client/hdfs/src/main/java/alluxio/hadoop/FileSystem.java

```diff
@@ -151,4 +157,37 @@ protected AlluxioURI getAlluxioPath(Path path) {
   protected Path getFsPath(String fsUriHeader, URIStatus fileStatus) {
     return new Path(fsUriHeader + fileStatus.getPath());
   }
+
+  @Override
+  public BlockLocation[] getFileBlockLocations(FileStatus file, long start, long len)
+      throws IOException {
+    if (mFileSystem.getConf().getBoolean(PropertyKey.DORA_ENABLED)) {
```

Comment:
Is it possible to make this value as a field of this class?  then we can initialize the value in the constructor or passed in from caller,  becuase reading a value from conf for every read would be expensive.

Reply:
I don't think we need the check.  dora will be always enabled